lib/models/Review.js

```diff
@@ -0,0 +1,48 @@
+const pool = require('../utils/pool');
+
+module.exports = class Review{
+  id;
+  stars;
+  detail;
+  user;
+
+  constructor(row) {
+    this.id = row.id;
+    this.stars = row.stars;
+    this.detail = row.detail;
+  }
+
+  static async insert({ stars, detail }) {
```

Comment:
Your insert should take in the stars, detail, user_id, and rest_id here to be inserted into your table directly. You shouldn't need to add the review by id if it is being added in this insert function.